test/framework/k8s_cluster.go

```diff
@@ -413,73 +418,6 @@ func (c *K8sCluster) GetKuma() ControlPlane {
 	return c.controlplane
 }
 
-func (c *K8sCluster) RestartKuma() error {
```

Comment:
Yeah, OK. That is from the old multi-zone e2e testing days. We are much more flexible now, so this is not needed.